app/Http/Controllers/QuizSubmissionController.php

```diff
@@ -17,4 +17,14 @@ public function show(QuizSubmission $quizSubmission): Response
 
         return Inertia::render("User/Quiz", ["submission" => QuizSubmissionResource::make($quizSubmission)]);
     }
+
+    public function result(QuizSubmission $quizSubmission): Response
+    {
+        $quizSubmission->load(["answerRecords.question.answers", "quiz"]);
+
+        return Inertia::render("User/QuizResult", [
```

Comment:
It leads to 404 bc page is named differently

Reply:
fixed